beta-prompts/beta_prompts/__init__.py

```
@@ -0,0 +1,208 @@
+# Package-level configuration with type hints
+DEFAULT_CONFIG: dict[str, str | int | bool] = {
+    "container_image": "python:3.11-slim",
+    "timeout_seconds": 300,
+    "max_retries": 3,
+    "api_timeout": 30,
+    "safety_enabled": True,
+    "logging_level": "INFO",
+}
+
+# Dynamic __all__ based on available components
+__all__ = (
```

Ruff (0.12.2)

Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)